apps/frontend/src/pages/friends/components/friend/friend.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button, Image } from "~/libs/components/components.js";
+import { type FriendDto } from "~/libs/types/types.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	user: FriendDto;
+};
+
+const Friend: React.FC<Properties> = ({
+	user: { fullName, imageUrl },
```

Comment:
```suggestion
	user,
```
avoid destruction in objects in props